plugins/onepad/CMakeLists.txt

```diff
@@ -22,7 +22,8 @@ set(onepadSources
 set(onepadHeaders
 	bitwise.h
 	controller.h
-	joystick.h
+	SDL/joystick.h
+    GamePad.h
```

Comment:
spaces here (and other places you added/changed) while tabs everywhere else in this file.